app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/common/DotsIndicator.kt

```diff
@@ -0,0 +1,52 @@
+package com.kks.nimblesurveyjetpackcompose.ui.presentation.common
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+// Reference https://medium.com/@prashantappdeveloper/viewpager-in-jetpack-compose-with-dot-indicators-within-minutes-a2779970534e
+@Suppress("LongParameterList")
```

Comment:
We should probably update our `detekt-rules` instead of suppressing this. Composables tend to have long parameter lists so adding the suppression every time might not be the best idea 🤔